src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -93,45 +63,68 @@
 import ch.njol.skript.util.ExceptionUtils;
 import ch.njol.skript.util.Task;
 import ch.njol.skript.variables.Variables;
-import ch.njol.util.Callback;
 import ch.njol.util.Kleenean;
 import ch.njol.util.NonNullPair;
 import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.Callable;
+import java.util.regex.Matcher;
 
 /**
  * @author Peter Güttinger
  */
 final public class ScriptLoader {
-	private ScriptLoader() {}
-	
+	private ScriptLoader() {
+	}
+
 	private final static Message m_no_errors = new Message("skript.no errors"),
 			m_no_scripts = new Message("skript.no scripts");
 	private final static PluralizingArgsMessage m_scripts_loaded = new PluralizingArgsMessage("skript.scripts loaded");
-	
+

```

Comment:
Seriously, no need to remove tabs in empty lines. Doesn't matter for readability.

Reply:
Ugh, this would be a big pain to try to revert that, is this change really very annoying? It affects literally every file

Reply:
That is why it is annoying; it affects all files. Before merging, someone will have to read through the whole set of changed files. Which means about 650 files... It is going to take me or @Snow-Pyon ages.

But yeah, we can do it. Will just take _some_ time :)